src/App.tsx

```diff
@@ -1,21 +1,29 @@
 import React from 'react';
 import logo from './logo.png';
+
+import { saveAs } from 'file-saver'
```

Comment:
Let's not change App.tsx. Everything should be tested in HavenoDaemon.test.ts.

Reply:
i'll rip it out after i'm done with the backend. i like to click and step through the code